### PR TITLE
Add testcase to verify ky-KG (Kyrgyz-Kyrgyzstan)

### PR DESCRIPTION
**Pull request by gouniLee**
### Checklist

* [ ] At least one test case is included for this feature or bug fix.
* [x] `ReleaseNotes` has added or is not needed.
* [x] This PR has passed all of the test cases on `Nodejs` and `Chrome Browser`.
* [x] This PR has passed all of the test cases on `QT/QML`.
* [ ] This is an API breaking change.
* [ ] Requires a major version change.

### Issue Resolved / Feature Added
* Regarding ky(Kyrgyz) language, the default script is `Cyrl` not `Arabic`. so I fixed cldr tool code (removed `ky` in rtl list)  and re-generate `dateformat.json` and `sysres.json`. (Removed rtl mark (200f) also fixed script order in ky/scripts.jf to `Cyrl` come first.
* Added test cases to verify `ky-KG` :  LocaleInfo,Script, String(plural) LocaleMatch, DateFmt, DurFmt, DateRngFmt, List, Unit, firstDayofWeek, clock, calendar,Month/Weekday Translation, NumFmt,
* timezone : Asia/Bishkek
* default calendar:  gregorian

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Links
[//]: # (Related issues, references)

### CODE REVIEW

File: js/data/locale/ky/dateformats.json [hunk 1]
```diff
@@ -3,40 +3,40 @@
         "order": "{date} {time}",
         "date": {
             "dmwy": {
-                "f": "‏yyyy-'ж'., d-MMMM, EEEE",
-                "l": "‏yyyy-'ж'., d-MMMM, EEE",
-                "m": "‏yyyy-'ж'., d-MMM, EE",
-                "s": "‏d/M/yy, E"
+                "f": "yyyy-'ж'., d-MMMM, EEEE",
```
Comment (ehoogerbeets) [inline js/data/locale/ky/dateformats.json:6]: I don't see what the difference is here. Even when I click "hide whitespace differences", the changes in this file keep showing up...

File: js/data/locale/ky/sysres.json [hunk 2]
```diff
@@ -187,48 +187,48 @@
...
-    "1#1 mon ago|#{num} mons ago": "one#‏{num} ай мурун|#‏{num} ай мурун",
-    "1#1 yr ago|#{num} yrs ago": "one#‏{num} жыл мурун|#‏{num} жыл мурун",
-    "#{num}s ago": "one#‏{num} сек. мурун|#‏{num} сек. мурун",
-    "#{num}mi ago": "one#‏{num} мүн. мурун|#‏{num} мүн. мурун",
-    "#{num}h ago": "one#‏{num} саат. мурун|#‏{num} саат. мурун",
-    "#{num}d ago": "one#‏{num} күн мурун|#‏{num} күн мурун",
-    "#{num}w ago": "one#‏{num} апт. мурун|#‏{num} апт. мурун",
-    "#{num}mo ago": "one#‏{num} ай мурун|#‏{num} ай мурун",
-    "#{num}y ago": "one#‏{num} жыл мурун|#‏{num} жыл мурун",
+    "1#in 1 second|#in {num} seconds": "one#{num} секунддан кийин|#{num} секунддан кийин",
```
Comment (ehoogerbeets) [inline js/data/locale/ky/sysres.json:190]: Here either... what's the difference? Why are these files showing up here?

File: js/test/daterange/testdatefmtrange_ky_KG.js [hunk 3]
```diff
@@ -0,0 +1,692 @@
+/*
+ * testdatefmtrange_ky_KG.js - test the date range formatter object Georgian/Georgia
```
Comment (ehoogerbeets) [inline js/test/daterange/testdatefmtrange_ky_KG.js:2]: Kyrgyz/Kyrgyzstan

File: js/data/locale/ky/dateformats.json (hunk 1 above)
Comment (gouniLee) [reply js/data/locale/ky/dateformats.json:6]: The difference is `200f` rtl mark which is removed. When the tool generating data, if it's rtl, it added `200f` in front which is removed now in this data.

File: js/data/locale/ky/sysres.json (hunk 2 above)
Comment (gouniLee) [reply js/data/locale/ky/sysres.json:190]: The difference is `200f` rtl mark which is removed. When the tool generating data, if it's rtl, it added `200f` in front which is removed now in this data

File: js/data/locale/ky/dateformats.json (hunk 1 above)
Comment (ehoogerbeets) [reply js/data/locale/ky/dateformats.json:6]: I see... that char is not visible in the git diffs and it is there before this used to be RTL language. Now with the switch to Cyrillic, it is not considered RTL any more.

**Pull request merged**